index.js

```diff
@@ -15,6 +15,7 @@ global.AbortController = AbortController
     CosmJS
  */
 const mnemonic = process.env.MNEMONIC;
+const DENOM = process.env.DENOM;
```

Comment:
let's use lower camelCase for JavaScript variables for consistency

```suggestion
const denom = process.env.DENOM;
```